### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by taquitos**
Looks like GitHub wants this named differently 🤷‍♂️

### CODE REVIEW

**Comment by taquitos** [comment]
Sorry for the noise, GitHub did not like my filename.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 name: Bug report
 about: Filling a bug report
-title: ""
+title: ''
 labels: bug
-assignees: ""
+assignees: ''
```
Comment (taquitos) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: These changes came in from GitHub's issue template web ui. It looks fine to me.

Comment (aboedo) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: could you clarify? 
I do see the change to single-quote in git, are you referring to something else?

**Review by aboedo — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (taquitos) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: Nope, that was it. Just the single-quote change that GitHub did 😄

Comment (aboedo) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: ohhh I see, the changes were done by GH, I'd misunderstood 👍

**Pull request merged**